duckduckgo/iOS
# Autofill device passcode disabled screen

Task/Issue URL: https://app.asana.com/0/0/1203093081439357/f
Tech Design URL:
CC: @THISISDINOSAUR 

**Description**:
Implementation of the device passcode disabled screen + associated state management



**Steps to test this PR**:
1. This needs to be tested on device since simulator doesn't support disabling device passcode. In iOS Settings > Face ID and Passcode scroll down and select "Turn Passcode Off", which will also disable biometrics
2. Go to the Autofill Logins screen and you should be presented with the new screen
3. Re-enable device passcode and confirm that all functionality works as before

<img src="https://user-images.githubusercontent.com/91189922/205032204-f3264e7d-5021-47f2-86ce-4ad1412813ee.png" width=500/>



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
